src/main/webapp/WEB-INF/views/driver/all.jsp

```diff
@@ -0,0 +1,13 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Get all drivers</title>
+</head>
+    <body>
+    <c:forEach var="driver" items="${drivers}">
+        <option value="${driver.name}">${driver.name}</option>
+        <option value="${driver.licenseNumber}">${driver.licenseNumber}</option>
```

Comment:
can you please show how does it look in browser?
maybe let's put the data in a table?